demo/addons/godot-openxr/plugin.cfg

```diff
@@ -0,0 +1,7 @@
+[plugin]
+
+name="Godot OpenXR"
+description="Godot OpenXR plugin"
+author="Christoph Haagch and Bastiaan Olij"
```

Comment:
The name is Christoph Haag, ch are the initials :)

Reply:
I never knew that :)